agents/acu/acu_agent.py

```diff
@@ -974,12 +991,22 @@ def generate_scan(self, session, params):
         az_speed = params.get('az_speed')
         acc = params.get('acc')
         el_endpoint1 = params.get('el_endpoint1')
-        el_endpoint2 = params.get('el_endpoint2')
-        el_speed = params.get('el_speed')
+        el_endpoint2 = params.get('el_endpoint2', el_endpoint1)
+        el_speed = params.get('el_speed', 0.0)
+        num_batches = params.get('num_batches', None)
+        start_time = params.get('start_time', None)
+        wait_to_start = params.get('wait_to_start', 3)
+        step_time = params.get('step_time', 0.1)
+        batch_size = params.get('batch_size', 500)
+        az_start = params.get('az_start', 'mid_inc')
 
         yield self.acu_control.stop()
         g = sh.generate_constant_velocity_scan(az_endpoint1, az_endpoint2,
-                                               az_speed, acc, el_endpoint1, el_endpoint2, el_speed)
+                                               az_speed, acc, el_endpoint1,
+                                               el_endpoint2, el_speed,
+                                               num_batches, start_time,
+                                               wait_to_start, step_time,
+                                               batch_size, az_start)
```

Comment:
It is best practice, in python, to pass a function's keyword arguments as keyword arguments rather than by position.  I.e. `num_batches=num_batches`, etc.  This may seem verbose, but it saves trouble later if the interface needs to be altered.

That ^^^, above, is my request for change.

This stuff below vvv is an alternative solution for you to consider.

In cases where you're just passing through a bunch of things from another function, you can bundle up the keyword args in a dict, and pass them in using **.  For example:
```
  # Populate params dict.
  scan_args = {
    'num_batches': params.get('num_batches', None),
    'start_time': params.get('start_time', None),
    ...
  }

  ...
  g = sh.generate_constant_velocity_scan(..., **scan_params)
```

Note that when you do it this way, you can fairly easy recreate the case where you _don't specify_ a parameter at all ... if you want the default values to be chosen by sh.generate_constant_velocity_scan, you can do something like:
```
    # Put only the non-None args into scan_args
    scan_params = {k: params.get(k) for k in ['num_batches', 'start_time', ...]
                   if params.get(k) is not None}
    ...

    g = sh.generate_constant_velocity_scan(..., **scan_params)
```